app/assets/stylesheets/devices.scss

```diff
@@ -121,12 +122,12 @@ subhead {
 }
 
 .ui-logo-wacubo {
-  top: -51px;
+  top: -90px;
   left: 54px;
   position: absolute;
-  background: image-url('wacubo-logo.jpg') 0 0 / 100% 100%;
+  background: image-url('wacubo-logo-2018.jpg') 0 0 / 100% 100%;
   width: 180px;
-  height: 165px;
+  height: 230px;
```

Comment:
Got this value from PR #159 which changed the WACUBO logo shape from rectangle to more of a square. This year's logo is rectangle again so we had to make some adjustments so it would fit and be centered.

Reply:
Can you add a comment to the code itself to help the next dev that works on this?